kubectl-plugin/pkg/util/generation/generation.go

```diff
@@ -98,6 +99,18 @@ func (rayClusterSpecObject *RayClusterSpecObject) generateRayClusterSpec() *rayv
 								corev1.ResourceMemory: resource.MustParse(rayClusterSpecObject.WorkerMemory),
 							}))))))
 
+	gpuResource := resource.MustParse(rayClusterSpecObject.WorkerGPU)
+	if !gpuResource.IsZero() {
```

Comment:
Since we are adding the GPU option, should we add the option to specify the `gke-accelerator`? 
I do think this could also be done on a later date though.

Reply:
We probably want to generalize it further and support an option to specify node selectors